ui/src/app/pages/workspace/workspace-card.tsx

```diff
@@ -153,17 +155,24 @@ export const WorkspaceCard = fp.flow(withNavigation)(
       ]);
     }
 
-    onClick() {
+    trackWorkspaceNavigation() {
```

Comment:
please add:
* a comment describing what this does 
* `boolean` return value
* description of what the return value means

Reply:
Hopefully done